external/meta-python/recipes-devtools/python3-tensorrt/python3-tensorrt_7.1.3-1.bb

```diff
@@ -0,0 +1,56 @@
+SUMMARY = "Tensorrt Python bindings"
+
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = "file://python/packaging/LICENSE.txt;md5=c291e0a531e08d4914e269730ba2f70d"
+
+DEPENDS = "python3-pybind11 tensorrt"
+
+inherit setuptools3 cmake pkgconfig cuda
+
+SRC_REPO = "github.com/NVIDIA/TensorRT.git;protocol=https"
+SRCBRANCH = "master"
+SRC_URI = "git://${SRC_REPO};branch=${SRCBRANCH} \
```

Comment:
I think you can remove the `branch=` because will clone the repo by using the `master` branch by default
In this case, no need to define `SRCBRANCH`

Reply:
I know it's the default, but these are there to make it easier to override the repo location and branch via a bbappend or local.conf setting.

Reply:
Agree